src/silo/storage/pango_lineage_alias.test.cpp

```diff
@@ -1,23 +1,24 @@
-#include "silo/database.h"
-
 #include <gtest/gtest.h>
 #include <string>
 #include <unordered_map>
 
+#include "silo/storage/pango_lineage_alias.h"
+
 struct TestParameter {
    std::string input;
    std::string expected_result;
 };
 
 class ResolveAliasTestFixture : public ::testing::TestWithParam<TestParameter> {
   protected:
-   const std::unordered_map<std::string, std::string> alias_map = {{"X", "A"}, {"XY", "A.1"}};
+   const silo::PangoLineageAliasLookup alias_map =
```

Comment:
I honestly don't know, why this line works, but I would suggest no implicit casting: 
`const silo::PangoLineageAliasLookup alias_map = silo::PangoLineageAliasLookup(
      std::unordered_map<std::string, std::string>{{"X", "A"}, {"XY", "A.1"}}
   );`